next.config.js

```diff
@@ -1,52 +1,17 @@
-module.exports = {
+/* eslint-disable @typescript-eslint/no-var-requires */
+const withBundleAnalyzer = require('@next/bundle-analyzer')({
+  enabled: process.env.ANALYZE === 'true',
+});
+
+module.exports = withBundleAnalyzer({
   reactStrictMode: true,
   async redirects() {
     return [
       {
-        source: '/chain/:slug*',
-        destination: '/:slug*',
-        permanent: true,
-      },
-      {
-        source: '/chain/0x13881/:slug*',
-        destination: '/mumbai/:slug*',
-        permanent: true,
-      },
-      {
-        source: '/chain/0x89/:slug*',
-        destination: '/polygon/:slug*',
-        permanent: true,
-      },
-      {
-        source: '/chain/0x64/:slug*',
-        destination: '/gnosis/:slug*',
-        permanent: true,
-      },
-      {
-        source: '/chain/0x5/:slug*',
-        destination: '/goerli/:slug*',
-        permanent: true,
-      },
-      {
-        source: '/0x13881/:slug*',
-        destination: '/mumbai/:slug*',
-        permanent: true,
-      },
-      {
-        source: '/0x89/:slug*',
-        destination: '/polygon/:slug*',
-        permanent: true,
-      },
-      {
-        source: '/0x64/:slug*',
-        destination: '/gnosis/:slug*',
-        permanent: true,
-      },
-      {
-        source: '/0x5/:slug*',
-        destination: '/goerli/:slug*',
```

Comment:
maybe you could leave this. it costs us nothing to leave it, but we might prevent some old links breaking

Reply:
@vidvidvid i removed them cause we will be moving to a new domain so that won't work..

Reply:
ohh yeah i see. we'll just have to make sure to change the `NEXT_PUBLIC_QUESTCHAINS_URL` and the plausible `data-domain` when we migrate ✌️